_includes/doc/admin-guide/options/source-flags.md

```diff
@@ -28,15 +28,15 @@
     | LOG_NOTICE priority if not specified otherwise.
 
 - *no-header*: The no-header flag triggers {{ site.product.short_name }} to parse only
-    the PRI field of incoming messages, and put the rest of the message
+    the `PRI` field of incoming messages, and put the rest of the message
     contents into ${MSG}.
 
     Its functionality is similar to that of the no-parse flag, except
-    the no-header flag does not skip the PRI field.
+    the no-header flag does not skip the `PRI` field.
 
     **NOTE:** Essentially, the no-header flag signals {{ site.product.short_name }} that the
     syslog header is not present (or does not adhere to the conventions
-    / RFCs), so the entire message (except from the PRI field) is put
+    / RFCs), so the entire message (except from the `PRI` field) is put
     into ${MSG}.
```

Comment:
isn' ${MSG} the field and PRI the macro?

Like `(except the value from the ${PRI} macro) is put into the `MSG` field.` ?